wandb/sdk/wandb_init.py

```diff
@@ -454,7 +460,7 @@ def init(self):
             run._set_run_obj(ret.run)
         elif s.mode in ("offline", "dryrun"):
             backend.interface.send_run(run)
-        elif s.mode in ("async", "run"):
+        elif s.mode in ("async", "run", "mock"):
```

Comment:
remove this.   I am trying to get rid of this mode string altogether in favor of separate boolean flags